src/Persistence/MassTransit.EntityFrameworkCoreIntegration/Saga/EntityFrameworkSagaRepository.cs

```diff
@@ -89,7 +88,7 @@ void IProbeSite.Probe(ProbeContext context)
                         // Hack for locking row for the duration of the transaction.
                         var tableName = dbContext.GetTableName<TSaga>();
                         await dbContext.Database.ExecuteSqlCommandAsync(
-                            $"select 1 from {tableName} WITH (UPDLOCK, ROWLOCK) WHERE CorrelationId = @p0", CancellationToken.None, sagaId)
```

Comment:
I'll accept this, but wish we could pass the CancellationToken from the `ConsumeContext` to the database calls, in case the transport cancels the message.